multimedia/mjpg-streamer/files/mjpg-streamer.init

```diff
@@ -67,14 +67,18 @@ start_instance() {
 		output_arg="output_http.so"
 
 		config_get port "$s" 'port'
-		[ -n "$port" ] && output_arg="${output_arg} --port $port"
+		[ -n "$port" ] && output_arg="${output_arg} -p $port"
+
+		config_get listen_ip "$s" 'listen_ip'
+		[ -n "$listen_ip" ] && output_arg="${output_arg} -l $listen_ip"
 
 		config_get www "$s" 'www'
-		[ -n "$www" ] && output_arg="${output_arg} --www $www"
+		[ -n "$www" ] && output_arg="${output_arg} -w $www"
 
 		config_get username "$s" 'username'
 		config_get password "$s" 'password'
-		[ -n "$username" ] && [ -n "$password" ] && output_arg="${output_arg} --credentials $username:$password"
+
+		[ -n "$username" ] && [ -n "$password" ] && output_arg="${output_arg} -c $username:$password"
```

Comment:
My preference is to use the long (and un-ambiguous) names

Reply:
Changed back